index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+The seminars can be delivered in person at your office, via a video call or a hybrid approach.
+
+In addition to the off-the-shelf course, I am happy to tailor the course to _match your needs, experience and technologies_.
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to craft a bespoke course for your business.**
+
+I also like to foster an on-going relationship after you have completed the course to _help you put these tools and techniques into practice_.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn the skills needed for _effective software development_.
 
+### Source Control
 
-### Souce Control
+Source control is a tool that stores your source code, and tracks changes made to it across time. It facilitates collaboration between multiple developers working on the same code, as well as ensuring reproducible results.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+**Source control is indispensable for working with other developers on code.**
+
+If you already use source control then this seminar can be tailored to whatever tools you are already using. I will cover best practices for using them successfully.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn how to use source control effectively.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+* Why you should use source control
+* How to setup a new Git repository
+* Basic operations of Git, using the command line and a GUI
+* Resolving merge conflicts
+* How to use the "Github workflow" to enhance collaboration
+
+Software & Technologies:
+ * [Git](https://git-scm.com/){:target="_blank"} - the industry standard source control solution
+ * [Gitlab](https://about.gitlab.com/){:target="_blank"} - a widely used cloud tool for storing source code and more
+ * [Git-Cola](https://git-cola.github.io/){:target="_blank"} - a user friendly tool for interacting with Git
+ * [Kdiff3](https://kdiff3.sourceforge.net/){:target="_blank"} - a user friendly tool for resolving merge conflicts
+
+This seminar can be tweaked to use different source control software, hosting options to match your existing workflows.  Please get in touch with me and we can customise this talk to meet your needs and match your skills.
```

Comment:
"different source control software, hosting options" <- Should the comma be an "and"? Otherwise I don't understand this sentence.